easybuild/easyconfigs/h/HEASoft/HEASoft-6.27.2-GCCcore-4.9.3.eb

```diff
@@ -0,0 +1,89 @@
+name = 'HEASoft'
+version = '6.27.2'
+
+
+dependencies = [
+    ( 'CFITSIO', '3.48' ),
+    ( 'Perl', '5.30.2' ),
+]
+
+easyblock = 'ConfigureMake'
+
+homepage = ''
+description = """"""
+
+toolchain = {'name': 'GCCcore', 'version': '4.9.3'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'https://www.isdc.unige.ch/~savchenk/cache/heasoft/',
+    'https://heasarc.gsfc.nasa.gov/FTP/software/lheasoft/lheasoft%(version)s/',
+]
+sources = ['heasoft-%(version)ssrc_no_xspec_modeldata.tar.gz']
+patches = []
+
+preconfigopts = "cd BUILD_DIR/ &&"
+prebuildopts = "cd BUILD_DIR/ &&"
+
+# build fails with parallel
+parallel = False
+
+preinstallopts = "cd BUILD_DIR/ &&"
+
+# note that "x86_64-pc-linux-gnu-libc2.17" is hardcoded, since it is derived from toolchain and platform
+# it might differ in different case: to be checked
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['x86_64-pc-linux-gnu-libc2.17/bin', 'x86_64-pc-linux-gnu-libc2.17/lib'],
```

Comment:
It would be good to specify a few key binaries and/or libraries in the file section here. dirs is less important (since it's usually caught by `files`

Reply:
sure, added some `files`: [there](https://github.com/volodymyrss/easybuild-easyconfigs/blob/c23511f9a6e4be4390db483d8fa185d1af4e2794/easybuild/easyconfigs/h/HEASoft/HEASoft-6.29-GCCcore-9.3.0.eb#L52-L55)